linelist_investigations/report_sources/temporal_trends_2019-10-17.Rmd

```diff
@@ -1269,6 +1270,366 @@ table_community_deaths_top_zones_time %>%
 
 
 
+<!-- ======================================================= -->
+<!-- ======================================================= -->
+<!-- ======================================================= -->
+# Nosocomial deaths {.tabset .tabset-fade .tabset-pills}
+
+<!-- ======================================================= -->
+## Outline 
+
+We use the variable `place_death_crude` which is derived from the 
+variable `place_of_death` to estimate the proportions of cases
+who are known to have died in a health facility (which is a subset
+of the community or non-CTE deaths):
+
+* overall, weekly 
+* by sub-coordination, monthly
+* for the main recently active health zones, monthly
+
+Each analysis will present:
+
+* overall results aggregated for the whole time period
+* a temporal breakdown
+
+For each proportion, we derive the exact binomial confidence interval.
+
+
+
+<!-- ======================================================= -->
+## Note
+
+We will rely on `adorn_totals` for computing totals. This approach needs to take into account an
+[issue](https://github.com/sfirke/janitor/issues/285) with `adorn_totals`, which
+ignores the first column of data assuming it is a grouping factor. This will not
+be a problem for further stratification, where the function can be used safely.
+
+
+
+<!-- ======================================================= -->
+## Overall
+
+The analysis proceeds with the following steps:
+
+1. calculate overall numbers and corresponding proportions and confidence
+intervals
+
+2. calculate weekly numbers and corresponding proportions and confidence
+intervals
+
+3. plot temporal trends
+
+4. create a table from 1, renaming columns and displaying it 
+
+5. create a table from 2, renaming columns and displaying it 
+
+
+```{r nosocomial_deaths_overall}
+
+## step 1
+nosocomial_deaths_overall <- x %>%
+  count(place_death_crude) %>%
+  spread(place_death_crude, n, fill = 0, drop = FALSE) %>%
+  mutate(total = nrow(x), # avoid adorn_totals
+         denom = total - unknown,

```

Comment:
note: the denominator here excludes unknown place of death; this is not the case later